inc/saiacl.h

```diff
@@ -1046,6 +1046,23 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
+    /**
+     * @brief Number of used entries for all pipes
+     *        1st entry in the list points to pipe-0,next points to pipe-1 and so on.
+     * @type sai_u32_list_t
+     * @flags READ_ONLY
+     */
+    SAI_ACL_TABLE_ATTR_USED_ACL_ENTRY_LIST,
+
+    /**
+     * @brief Number of free entry space available in
+     *        the current table for all pipes
+     *        1st entry in the list points to pipe-0,next points to pipe-1 and so on.
+     * @type sai_u32_list_t
+     * @flags READ_ONLY
+     */
+    SAI_ACL_TABLE_ATTR_AVAILABLE_ACL_ENTRY_LIST,
```

Comment:
a table can be fixed size (SAI_ACL_TABLE_ATTR_SIZE = certain size) or variable size (SAI_ACL_TABLE_ATTR_SIZE = 0)
Can you please explain the usage in both cases